package.json

```diff
@@ -6,13 +6,29 @@
     "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
   },
   "main": "index.js",
+  "types": "./types/index.d.ts",
   "files": [
     "configs.js",
-    "lib"
+    "lib",
+    "types"
   ],
-	"exports": {
-		"./configs": "./configs.js",
-		".": "./index.js"
+  "exports": {
+    "./package.json": "./package.json",
+    "./configs": {
+      "types": "./types/configs.d.ts",
+      "default": "./configs.js"
+    },
+    ".": {
+      "types": "./types/index.d.ts",
+      "default": "./index.js"
+    }
+  },
+  "typesVersions": {
+    "*": {
+      "configs": [
+        "./types/configs.d.ts"
+      ]
+    }
```

Comment:
We already have `configs` types defined, so I don't think it's necessary to declare them again in `typesVersions`?

Reply:
Without it `Node10` module resolution will fail when importing from `@eslint-community/eslint-plugin-eslint-comments/configs`.